packages/react-native-tab-view/src/TabBar.tsx

```diff
@@ -82,13 +83,50 @@ const getFlattenedTabWidth = (style: StyleProp<ViewStyle>) => {
   return tabStyle?.width;
 };
 
+const getFlattenedPaddingLeft = (style: StyleProp<ViewStyle>) => {
+  const flattenStyle = StyleSheet.flatten(style);
+
+  return flattenStyle
+    ? flattenStyle.paddingLeft || flattenStyle.paddingHorizontal || 0
+    : 0;
+};
+
+const getFlattenedPaddingRight = (style: StyleProp<ViewStyle>) => {
+  const flattenStyle = StyleSheet.flatten(style);
+
+  return flattenStyle
+    ? flattenStyle.paddingRight || flattenStyle.paddingHorizontal || 0
+    : 0;
+};
```

Comment:
The point of `getFlattenedPadding(Left|Right)` is to retrieve the "real" padding and not only the style prop value.

So I'll prefer to keep those functions and use `getFlattenedStyleProperty` inside:
```js
const getFlattenedPaddingLeft = (style: StyleProp<ViewStyle>) => {
  return getFlattenedStyleProperty('paddingLeft', style) || getFlattenedStyleProperty('paddingHorizontal', style) || 0;
};
```

Reply:
I'm thinking… there is a downside of `getFlattenedStyleProperty` : There will be more call of `StyleSheet.flatten` and it can tax in terms of optimizations https://reactnative.dev/docs/stylesheet#flatten